source/blog/2016-10-07-announcing-euwe-1-beta1.html.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Announcing Euwe Beta1 
+author: jprause
+date: 2016-10-10 12:49:24 UTC
+comments: true
+published: true
+tags: releases announcements
+---
+
+On behalf of the ManageIQ team, we're pleased to announce the Beta release for ManageIQ Euwe! 
+
+Euwe is the next milestone release for ManageIQ cloud and virtualization management platform. The update is packed with new features and improvements. Highlights include:
+
+* Centralized Administration
```

Comment:
* Darga timelines have been replaced by PatternFly timelines in Euwe.  

* Notification drawer has been added using ActionCable with PatternFly's notification drawer powered by Angular.